bin/checkver.ps1

```diff
@@ -1,4 +1,4 @@
-if(!$env:SCOOP_HOME) { $env:SCOOP_HOME = resolve-path (split-path (split-path (scoop which scoop))) }
+if(!$env:SCOOP_HOME) { write-host 'ANO'; $env:SCOOP_HOME = scoop prefix scoop | Resolve-Path }

```

Comment:
What `ANO` for?

Reply:
Sorry. Debug :D

Reply:
Debug remainings

Will remove when I have time